sdk/resourcemanager/Azure.ResourceManager/src/ArmClientOptions.cs

```diff
@@ -29,6 +29,8 @@ public sealed class ArmClientOptions : ClientOptions
         /// Sets the api version to use for a given resource type.
         /// To find which API Versions are available in your environment you can use the <see cref="ResourceProviderResource.Get"/> method
         /// for the provider namespace you are interested in.
+        /// To find which <see cref="ResourceType"/> to use you can refer to the corresponding Resource in the documentation or XML documentation comments of the API
+        /// and then get its resource type from the Resource's ResourceType field.
```

Comment:
Is this better:
To determine the appropriate <see cref="ResourceType"/>, you can refer to the corresponding documentation or XML documentation comments of the API. Then, get its resource type from the Resource's ResourceType field.

Reply:
i think it might be a little different style against existing comment. I was trying to make it similar to the existing comment for API Versions above...

Reply:
sure, and move it to xml doc comment to make it more clear in intellisense. thx